### PR TITLE
doc: Add v0.5.0 RN.

### PR DESCRIPTION
**Pull request by qianouyang**
* Add v0.5.0 release notes.
Signed-off-by: qouyang <qian.ouyang@intel.com>

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+  * [Legal Disclaimers](#legal-disclaimers)
+
+
+## New in this Release v0.5.0
+
+Edge Conductor Release 0.5.0 includes the following:
+
+*  Declaratively node joining.
+
+*  Replace all uses of the term "experience kit" with "package/kit"
```
Comment (yhu6) [inline RELEASE-NOTES.md:30]:
```suggestion
*  Replace "experience kit" terms by "package" or "kit"
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+  * [Where to find the release v0.5.0](#where-to-find-the-release-v050)
+  * [Hardware and Software Requirements v0.5.0](#hardware-and-software-requirements-v050)
+  * [Legal Disclaimers](#legal-disclaimers)
+
+
+## New in this Release v0.5.0
+
+Edge Conductor Release 0.5.0 includes the following:
+
+*  Declaratively node joining.
```
Comment (yhu6) [inline RELEASE-NOTES.md:28]:
```suggestion
*  Support declarative node joining.
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Replace all uses of the term "experience kit" with "package/kit"
+
+*  Significantly reduce build and deployment time for EC cluster and service deployment
```
Comment (yhu6) [inline RELEASE-NOTES.md:32]:
```suggestion
*  Reduce the time for EC cluster build and service deployment
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Upgrade latest version of supported Cluster for v0.5.0 (Need a list)
```
Comment (yhu6) [inline RELEASE-NOTES.md:34]:
```suggestion
*  Upgrade to the latest versions on cluster  infra components:(Need a list)
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Upgrade supported services for V0.5.0 （Need a list)
```
Comment (yhu6) [inline RELEASE-NOTES.md:36]:
```suggestion
*  Upgrade to the latest versions on supported services: (Need a list)
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Enable Debian 11.3 OS support
```
Comment (yhu6) [inline RELEASE-NOTES.md:38]: as the host OS for all cluster nodes?

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Enable Ubuntu 22.04 OS & preempt_RT support
```
Comment (yhu6) [inline RELEASE-NOTES.md:40]:
```suggestion
*  Support Edge node with Ubuntu 22.04 as host OS & preempt_RT
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Add Cert-Management support
```
Comment (yhu6) [inline RELEASE-NOTES.md:42]:
```suggestion
*  integrated "cert-Management" service from upstream
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Enable error index for the error message
```
Comment (yhu6) [inline RELEASE-NOTES.md:44]:
```suggestion
*  Enable error code and indexing for the error messages
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Add 3 tutorials of "Hello Cluster Helm", "Web-info" and "RKE deployment"
```
Comment (yhu6) [inline RELEASE-NOTES.md:46]:
```suggestion
*  Add 3 user tutorial documents: "Hello Cluster Helm", "Web-info" and "RKE deployment"
```

File: RELEASE-NOTES.md (hunk 6 above)
Comment (qianouyang) [reply RELEASE-NOTES.md:38]: as host os.

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+## New in this Release v0.5.0
+
+Edge Conductor Release 0.5.0 includes the following:
+
+*  Support declarative node joining.
+
+*  Replace "experience kit" terms by "package" or "kit"
+
+*  Reduce the time for EC cluster build and service deployment
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:32]:
```suggestion
*  Reduce the time for Edge Conductor cluster build and service deployment
```

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Upgrade to the latest versions on cluster  infra components:(Need a list)
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:34]:
```suggestion
*  Upgrade to the latest versions on cluster infrastructure components:(Need a list)
```

Comment (jnadelbx) [inline RELEASE-NOTES.md:34]: Should remove "Need a list" or put a list in?

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Upgrade to the latest versions on supported services: (Need a list)
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:36]: Remove "Need a list" or put a list in?

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Enable Debian 11.3 OS support
+
+*  Support Edge node with Ubuntu 22.04 as host OS & preempt_RT
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:40]:
```suggestion
*  Support edge node with Ubuntu 22.04 as host OS & preempt_RT
```

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  integrated "cert-Management" service from upstream
+
+*  Enable error code and indexing for the error messages
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:44]:
```suggestion
*  Enable error code and indexing for error messages
```

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+*  Add 3 user tutorial documents: "Hello Cluster Helm", "Web-info" and "RKE deployment"
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:46]:
```suggestion
*  Add 3 user tutorial documents: "Hello Cluster Helm," "Web-info," and "RKE deployment"
```

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+**EPJ-2755**
+
+Description: The system time of RKE node is inconsistent with the time of day-0 node.
+
+Workaround: No workaround, it will be fixed in next release.
+  
+
+**EPJ-2007**
+
+Description:
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:97]: Put in description.

File: RELEASE-NOTES.md [hunk 18]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+
+**EPJ-2007**
+
+Description: 
+
+Workaround: 
+**EPJ-2126**
+
+Description: When using ESP as the OS provider to profile the OS of the
+nodes, the user name and password is not set from the Package Kit
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:103]:
```suggestion
nodes, the user name and password is not set from the package kit
```

File: RELEASE-NOTES.md [hunk 19]
```diff
@@ -5,6 +5,195 @@ issues and limitations, and legal information. To learn more, refer to the
...
+config file, but set from the ESP profile.
+
+Workaround: Use the user name and password set in the official ESP profile to
+config the node list in a Package Kit config.
+
+**EPJ-2376**
+
+Description: When using CAPI deploy cluster with crio as container runtime, there is error message "overlayfs: unrecognized mount option "volatile" or missing value" on the provisioned nodes. This message is by design of the upstream project. Refer to the [known issue](https://github.com/cri-o/cri-o/issues/4773) of CRI-O.
```
Comment (jnadelbx) [inline RELEASE-NOTES.md:111]:
```suggestion
Description: When using CAPI deploy cluster with crio as container runtime, the error message "overlayfs: unrecognized mount option "volatile" or missing value" occurs on the provisioned nodes. This message is by design of the upstream project. Refer to the [known issue](https://github.com/cri-o/cri-o/issues/4773) of CRI-O.
```

**Review by jnadelbx — CHANGES_REQUESTED** [review]
Doc reviewed changes requested.

File: RELEASE-NOTES.md (hunk 12 above)
Comment (qianouyang) [reply RELEASE-NOTES.md]: Removed, the List is under the feature & version table.

File: RELEASE-NOTES.md (hunk 13 above)
Comment (qianouyang) [reply RELEASE-NOTES.md]: Removed. Same as above.

**Review by sunausti — APPROVED** [review]

**Review by jnadelbx — APPROVED** [review]

**Review by ychen2u — APPROVED** [review]

**Pull request merged**